plugins/jobs/girder_jobs/models/job.py

```diff
@@ -248,7 +233,7 @@ def createJob(self, title, type, args=(), kwargs=None, user=None, when=None,
             'log': [],
             'meta': {},
             'handler': handler,
-            'async_': async_,
+            'async_': asynchronous,
```

Comment:
Is there a reason we can't name this model property `'asynchronous'` too, so we have consistency?

Reply:
I was trying to avoid yet another check [like this](https://github.com/girder/girder/blob/master/plugins/jobs/girder_jobs/models/job.py#L327), but I guess it probably doesn't matter. I'll change it in the database.

Reply:
I think we can omit the check for `'async_'` entirely, since that field only exists within 3.0 pre-release instances. The check for `'async'` is perhaps more reasonable, to accommodate still-pending jobs in a 2.x -> 3 upgrade, but given the fact that `scheduleJob` is typically called immediately after creating a new Job document, I think we can safely remove that compatibility code too.

Reply:
I changed it, I kept in the `async` fallback check for now.